worlds/terraria/docs/setup_en.md

```diff
@@ -7,16 +7,28 @@ and [TModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Stea
 
 ## Installing the Archipelago Mod
 
-Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam.
+1. Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam
+2. Open TModLoader
+3. Go to `Workshop > Manage Mods` and enable the Archipelago mod
+   - If TModLoader states that you need version 1.4.3, follow the following steps
+     1. Close TModLoader
+     2. Right-Click TModLoader in Steam and select `Properties`
+     3. Navigate to `Betas -> Beta Participation`
```

Comment:
As before, "Betas" > "Beta Participation"

Reply:
And so on

Reply:
See https://github.com/ArchipelagoMW/Archipelago/pull/2149#discussion_r1320794135